{
	"dataType": "CVE_RECORD",
	"dataVersion": "5.1",
	"cveMetadata": {
		"cveId": "CVE-2022-1111",
		"assignerOrgId": "ceab7361-8a18-47b1-92ba-4d7d25f6715a",
		"assignerShortName": "GitLab",
		"state": "PUBLISHED",
		"datePublished": "2022-04-04T19:46:10",
		"dateUpdated": "2024-08-02T23:55:24.171Z",
		"dateReserved": "2022-03-28T00:00:00"
	},
	"containers": {
		"cna": {
			"providerMetadata": {
				"orgID": "ceab7361-8a18-47b1-92ba-4d7d25f6715a",
				"shortName": "GitLab",
				"dateUpdated": "2022-04-04T19:46:10"
			},
			"descriptions": [
				{
					"lang": "en",
					"value": "A business logic error in Project Import in GitLab CE/EE versions 14.9 prior to 14.9.2, 14.8 prior to 14.8.5, and 14.0 prior to 14.7.7 under certain conditions caused imported projects to show an incorrect user in the 'Access Granted' column in the project membership pages"
				}
			],
			"affected": [
				{
					"vendor": "GitLab",
					"product": "GitLab",
					"versions": [
						{
							"status": "affected",
							"version": ">=14.9, <14.9.2"
						},
						{
							"status": "affected",
							"version": ">=14.8.0, <14.8.5"
						},
						{
							"status": "affected",
							"version": ">=14.0, <14.7.7"
						}
					]
				}
			],
			"problemTypes": [
				{
					"descriptions": [
						{
							"type": "text",
							"lang": "en",
							"description": "Vulnerability in GitLab"
						}
					]
				}
			],
			"metrics": [
				{
					"cvssV3_1": {
						"version": "3.1",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:H/UI:R/S:U/C:N/I:L/A:N",
						"attackVector": "NETWORK",
						"attackComplexity": "LOW",
						"privilegesRequired": "HIGH",
						"userInteraction": "REQUIRED",
						"scope": "UNCHANGED",
						"confidentialityImpact": "NONE",
						"integrityImpact": "LOW",
						"availabilityImpact": "NONE",
						"baseScore": 2.4,
						"baseSeverity": "LOW"
					}
				}
			],
			"references": [
				{
					"tags": [
						"x_refsource_MISC"
					],
					"url": "https://gitlab.com/gitlab-org/gitlab/-/issues/345236"
				},
				{
					"tags": [
						"x_refsource_CONFIRM"
					],
					"url": "https://gitlab.com/gitlab-org/cves/-/blob/master/2022/CVE-2022-1111.json"
				}
			],
			"credits": [
				{
					"lang": "en",
					"value": "This vulnerability has been discovered internally by the GitLab team"
				}
			],
			"x_legacyV4Record": {
				"CVE_data_meta": {
					"ASSIGNER": "cve@gitlab.com",
					"ID": "CVE-2022-1111",
					"STATE": "PUBLIC"
				},
				"affects": {
					"vendor": {
						"vendor_data": [
							{
								"product": {
									"product_data": [
										{
											"product_name": "GitLab",
											"version": {
												"version_data": [
													{
														"version_value": ">=14.9, <14.9.2"
													},
													{
														"version_value": ">=14.8.0, <14.8.5"
													},
													{
														"version_value": ">=14.0, <14.7.7"
													}
												]
											}
										}
									]
								},
								"vendor_name": "GitLab"
							}
						]
					}
				},
				"credit": [
					{
						"lang": "eng",
						"value": "This vulnerability has been discovered internally by the GitLab team"
					}
				],
				"data_format": "MITRE",
				"data_type": "CVE",
				"data_version": "4.0",
				"description": {
					"description_data": [
						{
							"lang": "eng",
							"value": "A business logic error in Project Import in GitLab CE/EE versions 14.9 prior to 14.9.2, 14.8 prior to 14.8.5, and 14.0 prior to 14.7.7 under certain conditions caused imported projects to show an incorrect user in the 'Access Granted' column in the project membership pages"
						}
					]
				},
				"impact": {
					"cvss": {
						"attackComplexity": "LOW",
						"attackVector": "NETWORK",
						"availabilityImpact": "NONE",
						"baseScore": 2.3,
						"baseSeverity": "LOW",
						"confidentialityImpact": "NONE",
						"integrityImpact": "LOW",
						"privilegesRequired": "HIGH",
						"scope": "UNCHANGED",
						"userInteraction": "REQUIRED",
						"vectorString": "AV:N/AC:L/PR:H/UI:R/S:U/C:N/I:L/A:N",
						"version": "3.1"
					}
				},
				"problemtype": {
					"problemtype_data": [
						{
							"description": [
								{
									"lang": "eng",
									"value": "Vulnerability in GitLab"
								}
							]
						}
					]
				},
				"references": {
					"reference_data": [
						{
							"name": "https://gitlab.com/gitlab-org/gitlab/-/issues/345236",
							"refsource": "MISC",
							"url": "https://gitlab.com/gitlab-org/gitlab/-/issues/345236"
						},
						{
							"name": "https://gitlab.com/gitlab-org/cves/-/blob/master/2022/CVE-2022-1111.json",
							"refsource": "CONFIRM",
							"url": "https://gitlab.com/gitlab-org/cves/-/blob/master/2022/CVE-2022-1111.json"
						}
					]
				}
			}
		},
		"adp": [
			{
				"providerMetadata": {
					"orgID": "af854a3a-2127-422b-91ae-364da2661108",
					"shortName": "CVE",
					"dateUpdated": "2024-08-02T23:55:24.171Z"
				},
				"title": "CVE Program Container",
				"references": [
					{
						"tags": [
							"x_refsource_MISC",
							"x_transferred"
						],
						"url": "https://gitlab.com/gitlab-org/gitlab/-/issues/345236"
					},
					{
						"tags": [
							"x_refsource_CONFIRM",
							"x_transferred"
						],
						"url": "https://gitlab.com/gitlab-org/cves/-/blob/master/2022/CVE-2022-1111.json"
					}
				]
			}
		]
	}
}
